Dockerfile

```diff
@@ -1,12 +1,59 @@
+##
+# Build Stage
+#
+FROM maven:3-jdk-11 as build
+
+RUN apt-get -y update
+RUN apt-get -y install git apache2-utils
+
+##
+# Copy Source Code
+#
+ADD .git /build/.git
+ADD .gitmodules /build/.gitmodules
+ADD ./pom.xml /build/pom.xml
+ADD ./tools /build/tools
+ADD ./src /build/src
+ADD ./install /build/install
+ADD ./dev /build/dev
+
+WORKDIR /build
+
+##
+# Checkout Dependencies
+#
+RUN git submodule update --init --recursive
+
+ENV DEFAULT_PW="adminADMIN!"
+
+# OE Default Password
+RUN ./install/createDefaultPassword.sh -c -p ${DEFAULT_PW}
```

Comment:
@CalebSLane Is this the right place for the default password creation, during the build step? the password is saved into the resource directory and used in the build, correct?

Reply:
cc @mozzy11

Reply:
this should be the right place , before compiling the source code

what if we just run the non interactive command here directly without using the `createDefaultPassword.sh `
`htpasswd -b -c ./src/main/resources/adminPassword.txt admin ${DEFAULT_PW}`